tests/Microsoft.Bot.Configuration.Tests/ConfigurationLoadAndSaveTests.cs

```diff
@@ -84,6 +84,32 @@ public async Task CantLoadWithoutSecret()
             catch { }
         }
 
+        [TestMethod]
+        public async Task LoadFromFolderWithSecret()
```

Comment:
How about the "really" basic tests? 

Like: Load a file that doesn't exist. Make sure the right exception is thrown. 
Pass in null as the file name. Make sure exception. 
... etc.